.scalafmt.conf

```diff
@@ -0,0 +1,29 @@
+include ".scalafmt-common.conf" 
+
+// ScalaFmt stuff
```

Comment:
Unless you are intentionally overwriting common i do not think this is required
if you are are overwriting common then id be happy to change common so it meets your needs

Reply:
Sure, would you make the change to remove the contents of scalafmt.conf please?